### PR TITLE
Update documentation of Simulator::solve_stokes().

### PR DESCRIPTION
**Pull request by bangerth**
While there, also use an empty line to better delineate two parts of a
comment.

@mfraters: this is what we talked about last week. Can you take a look?
I didn't update comments in the solve_stokes() function itself so as not to
step onto the changes you make in #2070.

### CODE REVIEW

File: include/aspect/simulator.h [hunk 1]
```diff
@@ -611,12 +611,43 @@ namespace aspect
       /**
        * Solve the Stokes linear system. Return the initial nonlinear
        * residual, i.e., if the linear system to be solved is $Ax=b$, then
-       * return $\|Ax_0-b\|$ where $x_0$ is the initial guess for the solution
-       * variable and is taken from the current_linearization_point member
+       * return $\|Ax_k-b\|$ where $x_k$ is the initial guess for the solution
+       * variable and is taken from the @p current_linearization_point member
        * variable. For the purpose of this function, this residual is computed
-       * only the velocity and pressure equations (i.e., for the 2x2 block
+       * only rom the velocity and pressure equations (i.e., for the 2x2 block
```
Comment (MFraters) [inline include/aspect/simulator.h]: from?

Comment (bangerth) [reply include/aspect/simulator.h]: Yes, good point. Fixed!

**Comment by bangerth** [comment]
Hm. If you don't too violently disagree, can we merge this and I revisit what I write here once we've got #2070 sorted out?

**Comment by MFraters** [comment]
Sure, it is correct with respect to what it is currently doing :)

**Pull request merged**